bundles/org.openhab.binding.ism8/src/main/java/org/openhab/binding/ism8/server/DataPointBase.java

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ism8.server;
+
+import java.nio.ByteBuffer;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link DataPointBase} is the base class for all data points
+ *
+ * @author Hans-Reiner Hoffmann - Initial contribution
+ */
+@NonNullByDefault
+public abstract class DataPointBase<T> implements IDataPoint {
+
+    private final Logger logger = LoggerFactory.getLogger(DataPointBase.class);
+
+    private int __Id;
+    private String __KnxDataType = new String();
+    private String __Description = new String();
+    private T __Value;
+    private String __Unit = new String();
```

Comment:
Please make sure that all field and variable names are camelcase.

Reply:
Also, instead of `new String()` just use an empty string `""`

Reply:
Done